.github/workflows/create_llvm.yml

```diff
@@ -4,13 +4,19 @@ on:
   push:
     branches:
       - main
+    paths:
+      - '.github/workflows/create_llvm.yml'
+  schedule:
+    # Run every night at 19:00 hours
+    - cron: "0 19 * * *"
```

Comment:
it needs to run every so often or github deletes it.

Reply:
According to https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows, it does not, it is sufficient to have the other action run that merely accesses it: "GitHub will remove any cache entries that have not been accessed in over 7 days."

Reply:
true, but if we all go on holiday for a week then it would delete it.
Here's a counter proposal, we should be running the workflow that updates the pr cache (ideally on merge) - I could make that change instead and that would solve this issue.

Reply:
I'll take out the cron for now, but also the run when we merge part unless it's this file. We can look at other ways of ensuring something is run to keep llvm alive.